src/runtime/hexagon/rpc/simulator/session.cc

```diff
@@ -483,7 +483,13 @@ std::string SimulatorRPCChannel::Message_::str() const {
     case Message::kSendEnd:
       return "kSendEnd";
     default:
-      break;
+      LOG(FATAL) << "Internal error: unexpected message code: " << msg.code;
+#if (__GCC__ || __clang__)
+#if __has_builtin(__builtin_unreachable)
+      __builtin_unreachable()
+#endif
+#endif
+          break;
```

Comment:
Without this, gcc 7.5 was complaining about a code path (`default:`) with no return value.  Since this could be awful to debug if it happened, I don't want to simply suppress the warning.

Reply:
The `LOG(FATAL)` should terminate the program, so the builtin isn't really needed---more so that it only happens if the compiler supports it.  A better thing may be to return `"kInvalid_" + std::to_string(msg.code)` or something like that if you don't trust the LOG to cause an abort.